tests/tracing/static/simple.output.yaml

```diff
@@ -1,114 +0,0 @@
----
-# Source: sumologic/templates/otelcol-configmap.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: RELEASE-NAME-sumologic-otelcol
-  labels:
-    app: RELEASE-NAME-sumologic-fluentd-metrics
-    chart: "sumologic-1.1.0-beta.2"
-    release: "RELEASE-NAME"
-    heritage: "Helm"
-data:
-  traces.otelcol.conf.yaml: |-
```

Comment:
This is test for otelcol, we shouldn't remove it

Reply:
But these tests reference the configmap, which is now removed?

Reply:
it references otelcol configmap

Reply:
Ahh, got it.  Ok fixed this.